nikola/plugins/task/indexes.py

```diff
@@ -253,11 +261,17 @@ def index_section_path(self, name, lang, is_feed=False):
             index_file = os.path.splitext(self.site.config['INDEX_FILE'])[0] + extension
         else:
             index_file = self.site.config['INDEX_FILE']
+        if name in self.number_of_pages_section[lang]:
+            number_of_pages = self.number_of_pages_section[lang][name]
+        else:
+            posts = [post for post in self._get_filtered_posts(lang, self.site.config['SHOW_UNTRANSLATED_POSTS']) if post.section_slug(lang) == name]
+            number_of_pages = self._compute_number_of_pages(posts, self.site.config['INDEX_DISPLAY_POST_COUNT'])
+            self.number_of_pages_section[lang][name] = number_of_pages
         return utils.adjust_name_for_index_path_list([_f for _f in [self.site.config['TRANSLATIONS'][lang],
```

Comment:
Here, the arguments aren't needed for `utils.adjust_name_for_index_path_list` since for the given arguments, it simply returns the first argument. (In particular, computing the page numbers isn't needed for sections.)